Source/driver/Castro_advance_ctu.cpp

```diff
@@ -147,6 +147,14 @@ Castro::do_advance_ctu(Real time,
     }
 
 
+#ifdef SIMPLIFIED_SDC
+    // the SDC reactive source ghost cells on coarse levels might not
```

Comment:
Need to guard this also with #ifdef REACTIONS, I believe

Reply:
done